br/br-monitoring-and-alert.md

```diff
@@ -28,7 +28,7 @@ aliases: ['/zh/tidb/dev/pitr-monitoring-and-alert/']
 
 | 指标                                                    | 类型        | 说明                                                                                                                                              |
 |-------------------------------------------------------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------|
-| **tikv_log_backup_interal_actor_acting_duration_sec** | Histogram | 处理内部各种消息事件的耗时。<br/>`message :: TaskType`                                                                                                        |
+| **tikv_log_backup_internal_actor_acting_duration_sec** | Histogram | 处理内部各种消息事件的耗时。<br/>`message :: TaskType`                                                                                                        |
```

Comment:
This typo was from https://raw.githubusercontent.com/tikv/tikv/master/metrics/grafana/tikv_details.json, perhaps we need to fix it in the json file first.